data/localized/de_patch/text/game/abilities.stringtable

```diff
@@ -3571,32 +3571,32 @@
     </Entry>
     <Entry>
       <ID>731</ID>
-      <DefaultText>Bonus Gestaltwandler Bär</DefaultText>
+      <DefaultText>Bonus Gestaltwandler: Bär</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>732</ID>
-      <DefaultText>Bonus Gestaltwandler Eber</DefaultText>
+      <DefaultText>Bonus Gestaltwandler: Eber</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>733</ID>
-      <DefaultText>Bonus Gestaltwandler Kater</DefaultText>
+      <DefaultText>Bonus Gestaltwandler: Kater</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>734</ID>
-      <DefaultText>Bonus Gestaltwandler Wächter</DefaultText>
+      <DefaultText>Bonus Gestaltwandler: Wächter</DefaultText>
```

Comment:
In Russisch ist es eher ein Förster/ Waldwächter.
Das müsste also das Talent für eine zusätzliche Gestaltwandlung für den Waldläufer sein.